### PR TITLE
SDI 355 Info incentives properties

### PR DESCRIPTION
**Pull request by andymarke**
TODO - get current migration count and last migration

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visits/VisitMigrationProperties.kt [hunk 1]
```diff
@@ -18,17 +18,17 @@ class VisitMigrationProperties(
   private var visitMappingService: VisitMappingService
 ) : InfoContributor {
 
-  internal val registersQueue by lazy { hmppsQueueService.findByQueueId(VISITS_QUEUE_ID) as HmppsQueue }
+  internal val queue by lazy { hmppsQueueService.findByQueueId(VISITS_QUEUE_ID) as HmppsQueue }
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visits/VisitMigrationProperties.kt:21]: copy and paste typo in name

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import org.springframework.boot.actuate.info.Info.Builder
+import org.springframework.boot.actuate.info.InfoContributor
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.INCENTIVES_QUEUE_ID
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+
+@Component
+class IncentiveMigrationProperties(
+  private var hmppsQueueService: HmppsQueueService,
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt:17]: odd to be a `var` here?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import com.amazonaws.services.sqs.model.QueueAttributeName.ApproximateNumberOfMessagesNotVisible
+import org.springframework.boot.actuate.info.Info.Builder
+import org.springframework.boot.actuate.info.InfoContributor
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.INCENTIVES_QUEUE_ID
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+
+@Component
+class IncentiveMigrationProperties(
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt:16]: looks very similar to the visits one below, could be pulled out into superclass?

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt (hunk 3 above)
Comment (andymarke) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt:16]: Probably, not sure yet - depends when I do the TODO bits

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt (hunk 2 above)
Comment (andymarke) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMigrationProperties.kt:17]: yes, not sure what has happened there

**Pull request merged**